collections/_article/godotcon-2025-xr-android-recap.md

```diff
@@ -0,0 +1,135 @@
+You can see the full presentation slides at [https://bit.ly/godotcon-boston-2025-xr-android](https://bit.ly/godotcon-boston-2025-xr-android).
+
+# Android & XR editor
+
+## Android editor
+
+<img alt="Android editor on phone" src="/storage/blog/covers/godotcon-boston-2025-android-xr.webp" />
+
+An update on the Android editor is long overdue\! Since [its release in 2023](https://godotengine.org/article/android_godot_editor_play_store_beta_release/), the Android editor has been available on the [Google Play store](https://play.google.com/store/apps/details?id=org.godotengine.editor.v4) and on [Godot’s download page](https://godotengine.org/download/android/) with support for all Android devices (e.g.: phones, foldables, tablets, Chromebooks, and others). 
+
+Last fall, during GodotCon 2024, we announced that the Android editor had **\~850K device installs** from the Google Play store. Over the last 6 months, installs grew by another **\~365K** for a total of **1.2M device installs** by GodotCon (Boston) 2025\. This corresponds to over **\~2000+ daily installs** with over **\~90K monthly active users**.  
+
+<img alt="Android editor device installs 2025" src="/storage/blog/godotcon-2025-android-xr/godot-android-editor-device-install-2025.webp" />
+
+From the Google Play store data, we made a few interesting observations worth diving into.
+
+### Most installs are from Brazil \+ India
+
+Brazil and India make up **26%** of installs with **17%** coming from Brazil and **9%** coming from India. 
+
+<img alt="Android editor device installs percentage by country" src="/storage/blog/godotcon-2025-android-xr/godot-android-editor-countries-installs-2025.webp" />
+
+From this data point, we can observe how the Android editor is **helping to lower the barrier of entry to game dev** by allowing users in Brazil, India, and other countries to make use of **cheaper** and more **accessible** Android devices for game development.
+
+### Most installs are on phones\!
+
+According to the data, **\~87%** of installs **are on phones\!**   
+
+<img alt="Android editor device installs by form-factor percentage" src="/storage/blog/godotcon-2025-android-xr/godot-android-editor-device-type-percentage-2025.webp" />
+
+In the [release blog post](https://godotengine.org/article/android_godot_editor_play_store_beta_release/), we called out the Android editor as being optimized for large form-factor devices (e.g.: foldables, tablets, Chromebooks), so we expected those types of devices to be driving the installs. However, from the data it can be seen that tablets and Chromebooks only make up **\~10.6%** of installs. So while the experience is still unoptimized for phones, for a vast number of users, the Android editor has a valuable use and addresses a valid need on regular Android phones.
```

Comment:
```suggestion
In the [Android editor release blog post](https://godotengine.org/article/android_godot_editor_play_store_beta_release/), we mention it being optimized for large form-factor devices (e.g.: foldables, tablets, Chromebooks), so we expected those types of devices to be driving the installs. However, from the data it can be seen that tablets and Chromebooks only make up **\~10.6%** of installs. So while the experience is still unoptimized for phones, for a vast number of users, the Android editor has a valuable use and addresses a valid need on regular Android phones.
```

Adding some context here to specify which release blog post it is.